### PR TITLE
Spatial cv rev rl

### PR DESCRIPTION
**Pull request by Robinlovelace**

### CODE REVIEW

File: 13-spatial-cv.Rmd [hunk 1]
```diff
@@ -185,7 +218,7 @@ pred = raster::predict(object = ta, model = fit,
 ```
 
 
-```{r lsl-susc, echo = FALSE, fig.cap="Spatial prediction of landslide susceptibility using a GLM. CRS: UTM zone 17S (EPSG: 32717).", warning=FALSE}
+```{r lsl-susc, echo=FALSE, fig.cap="Spatial prediction of landslide susceptibility using a GLM. CRS: UTM zone 17S (EPSG: 32717).", warning=FALSE}
```
Comment (Robinlovelace) [inline 13-spatial-cv.Rmd]: Just a comment to say: this map looks AWESOME!

**Comment by Robinlovelace** [comment]
One question just occurred: why use **parallelMap** instead of other packages like https://github.com/HenrikBengtsson/future (which also has apply + lapply + map implementations).

File: 13-spatial-cv.Rmd [hunk 2]
```diff
@@ -151,30 +163,51 @@ tm_shape(hs, bbox = bbx) +
...
+pred_glm = predict(object = fit, type = "response")
+head(pred_glm)
 ```
 
-We can also predict spatially by applying the coefficients to our predictor rasters. 
-We could do this manually but can also use **raster**'s `predict()` function.
-Aside from the fitted model, this function also expects a raster stack with the predictors exactly named as in the fitted model (Figure \@ref(fig:lsl-susc)).
+Spatial predictions can be made by applying the coefficients to the predictor rasters. 
+This can be done manually or with `raster::predict()`.
+In addition to a model object (`fit`), this function also expects a raster stack with the predictors named in the fitted model formula (Figure \@ref(fig:lsl-susc)).
```
[the remark this answers is not in the document]
Comment (jannes-m) [reply 13-spatial-cv.Rmd]: It is in the exercises and the corresponding code in the vignette.Jakub suggested to put ta in spDataLarge, maybe the best solution?

**Comment by jannes-m** [comment]
mlr uses parallelMap internally, that's why we use it.

File: 13-spatial-cv.Rmd (hunk 2 above)
[the remark this answers is not in the document]
Comment (Robinlovelace) [reply 13-spatial-cv.Rmd]: Yes I think so - it's inconsistent currently as all other datasets are now either in **spData** or **spDataLarge** which is awesome.

**Comment by Robinlovelace** [comment]
OK this review is finally done, off to explore Nuremberg! Having read it I am even more inclined to mention modelling in the chapter title as it provides a good resource on modelling, although I can see now the focus is ml, spatial CV and hyperparameter tuning. How about the following suggestion?

# Predictive models, machine learning and spatial cross-validation

Happy to hear more concise suggestions and think 

# Spatial cross-validation for machine learning 

Is OK, just a little too specific.

**Comment by Robinlovelace** [comment]
The most concise and broad option I can think of is

# Modelling geographic data

Which I think would be OK if we broadened the intro and conclusions a little - would have a broader appeal but happy for something more specific - maybe an intermediate point on this `Predictive models, machine learning and spatial cross-validation -> Modelling geographic data` long+narrow-short+wide continuum would be optimal? Thoughts @jannes-m and @Nowosad ???

**Comment by jannes-m** [comment]
I have the impression that Modeling geographic data is rather general, what about **Statistical learning for spatial data**? This would also fit well with the definitions we are giving in the introduction.

**Comment by Robinlovelace** [comment]
:+1: except I'd say **Statistical learning for geographic data** to be consistent with the terminology in most of the book (not planning to do an analysis based on Martian data although that would be fun!).

**Comment by jannes-m** [comment]
Thanks for the many useful comments, suggestions and rewrites @Robinlovelace !!

**Comment by jannes-m** [comment]
good point!!

**Pull request merged**